lib/averages.js

```diff
@@ -0,0 +1,42 @@
+const total = (inputArray) => {
+  let sum = inputArray.reduce((sum, number) => {
```

Comment:
I would do one of two things here -
a) use a different variable name for either `sum` or the accumulator; as having two `sum` variables is a bit confusing.
b) cut a line out of this by returning this reduce on line 2 rather than line 6